app/views/autocomplete/show.erb

```diff
@@ -0,0 +1,3 @@
+<% @suggestions.each_with_index.map do |suggestion, i| %>
```

Comment:
Is this file used by anything?

Reply:
yes - it's how the autocomplete#show command takes the suggestions and serves them up the way the stimulus-autocomplete plugin needs.

Reply:
I see that conditional that would render this. Should it have an `html.erb` suffix?

Reply:
It didn't work with the html.erb suffix.

It's not html, it's an html fragement, so I typed it as "text".   If it were typed as html, but then I think we'd get the full html headers and such.

Reply:
Did you try passing `layout: false`?

Reply:
I don't understand - would `layout : false` be in the controller?   

Are you suggesting I put a method in the controller to do the reformatting?  Or that I add `layout : false` and then use `html.erb` as the suffix?   Maybe a realtime conversation would be better.

Reply:
Yes. You should have a `render layout: false` in the controller and then rename this with an `html.erb` suffix.